### DISCUSSION TITLE
How can i add third parties scripts in my Next.js v14.2.3?

### DISCUSSION THREAD

**Discussion by sarfarazsiddiquii** [vercel/next.js#68758] [Help]
### Summary

I have tried various approaches but i keep getting the same error.
 `(67791:1 Failed to execute 'write' on 'Document': It isn't possible to write into a document from an asynchronously-loaded external script unless it is explicitly opened.
(anonymous)@67791:1)`.

### Additional information

```js
Things i have tried - 

1.
<Head>
</Head>
<script src="//tags.orquideassp.com/--" />

2.
<Script
        src="//tags.orquideassp.com/--"
        strategy="--"
      />



3.
useEffect(() => {
    
    const script = document.createElement('script');
    script.src = "//tags.orquideassp.com/---";
    script.async = false; 
    script.defer = false; 
    document.head.appendChild(script);
    return () => {
      document.head.removeChild(script);
    };
  }, []);
```

**Comment by Moemen12** [comment]
see this maybe it helps
[stackoverflow](https://stackoverflow.com/questions/24297829/execute-write-on-doc-it-isnt-possible-to-write-into-a-document-from-an-asynchr)

**Reply by sarfarazsiddiquii** [reply]
Thanks, but I am not using `document.write()` anywhere, I am just simply adding a third-party script
